### PR TITLE
9 user profiles

### PR DESCRIPTION
**Pull request by SinGarTheGoat**

### CODE REVIEW

**Comment by SinGarTheGoat** [comment]
Im surprised it says they are clear to merge.

File: app/assets/stylesheets/application.scss [hunk 1]
```diff
@@ -14,6 +14,7 @@
 
  @import "bootstrap-sprockets";
  @import "bootstrap";
+ @import "bookmarks";
```
Comment (alexstophel) [inline app/assets/stylesheets/application.scss:17]: Add a new line after this import.

File: app/assets/stylesheets/bookmarks.scss [hunk 2]
```diff
@@ -1,10 +1,16 @@
...
 
 .url{
   margin-bottom: 0px;
 }
+
+.bookmark-detail-well {
+  margin-top: 10px;
+  word-wrap: break-word;
+
+  .btn {
```
Comment (alexstophel) [inline app/assets/stylesheets/bookmarks.scss]: Add some styles or remove this rule.

File: app/assets/stylesheets/users.scss [hunk 3]
```diff
@@ -0,0 +1,3 @@
+// Place all the styles related to the Users controller here.
```
Comment (alexstophel) [inline app/assets/stylesheets/users.scss]: You can get rid of this file for now.

File: app/helpers/users_helper.rb [hunk 4]
```diff
@@ -0,0 +1,2 @@
+module UsersHelper
```
Comment (alexstophel) [inline app/helpers/users_helper.rb]: You can delete this helper.

File: app/models/bookmark.rb [hunk 5]
```diff
@@ -5,6 +5,11 @@ class Bookmark < ActiveRecord::Base
   before_validation :httpset
   validates :url, format: { with: /\Ahttp:\/\/.*(com|org|net|gov)/i,
     message: "only allows valid URLs." }
+  scope :liked_by, ->(user) { joins(:likes).where(likes: { user_id: user.id }) }
+  # def self.liked_by(user)
```
Comment (alexstophel) [inline app/models/bookmark.rb]: Remove these 3 comments.

File: app/models/like.rb [hunk 6]
```diff
@@ -1,4 +1,5 @@
 class Like < ActiveRecord::Base
   belongs_to :bookmark
   belongs_to :user
+
```
Comment (alexstophel) [inline app/models/like.rb]: Remove this new line.

File: app/models/user.rb [hunk 7]
```diff
@@ -9,7 +9,7 @@ class User < ActiveRecord::Base
   has_many :topics
   has_many :likes, dependent: :destroy
 
-  def liked?(bookmark)
-    likes.where(bookmark_id: bookmark.id).any?
+  def liked(bookmark)
```
Comment (alexstophel) [inline app/models/user.rb:12]: Why this change?

File: config/routes.rb [hunk 8]
```diff
@@ -1,5 +1,7 @@
 Rails.application.routes.draw do
 
+  get 'users/show'
```
Comment (alexstophel) [inline config/routes.rb]: You don't need this anymore, right?

File: spec/helpers/users_helper_spec.rb [hunk 9]
```diff
@@ -0,0 +1,15 @@
+require 'rails_helper'
```
Comment (alexstophel) [inline spec/helpers/users_helper_spec.rb]: You can get rid of this file.

File: spec/views/users/show.html.erb_spec.rb [hunk 10]
```diff
@@ -0,0 +1,5 @@
+require 'rails_helper'
```
Comment (alexstophel) [inline spec/views/users/show.html.erb_spec.rb]: You can get rid of this file.

File: app/models/user.rb (hunk 7 above)
Comment (SinGarTheGoat) [inline app/models/user.rb:12]: I figure it is a boolean and this way I thought it read better
